src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -215,11 +215,23 @@ private void view_MouseRightButtonUp(object sender, MouseButtonEventArgs e)
 
         #endregion
 
-        public Ray3D GetClickRay(MouseEventArgs mouseButtonEventArgs)
+        private IRay GetClickRay(MouseEventArgs args)
```

Comment:
moved implementation from DM project to here to remove direct dependencies from DM on Helix.